eth/client.go

```diff
@@ -42,74 +47,76 @@ type LivepeerEthClient interface {
 	Backend() *ethclient.Client
 	Account() accounts.Account
 	SubscribeToJobEvent(ctx context.Context, logsCh chan types.Log) (ethereum.Subscription, error)
-	WatchEvent(logsCh <-chan types.Log) (types.Log, error)
-	RoundInfo() (*big.Int, *big.Int, *big.Int, *big.Int, error)
-	InitializeRound() (*types.Transaction, error)
+	RoundInfo() (*big.Int, *big.Int, *big.Int, error)
+	InitializeRound() (<-chan types.Receipt, <-chan error)
+	Transcoder(blockRewardCut uint8, feeShare uint8, pricePerSegment *big.Int) (<-chan types.Receipt, <-chan error)
+	Bond(amount *big.Int, toAddr common.Address) (<-chan types.Receipt, <-chan error)
+	Reward() (<-chan types.Receipt, <-chan error)
+	Job(streamId string, transcodingOptions string, maxPricePerSegment *big.Int) (<-chan types.Receipt, <-chan error)
+	ClaimWork(jobId *big.Int, segmentRange [2]*big.Int, claimRoot [32]byte) (<-chan types.Receipt, <-chan error)
+	Verify(jobId *big.Int, claimId *big.Int, segmentNumber *big.Int, dataHash string, transcodedDataHash string, broadcasterSig []byte, proof []byte) (<-chan types.Receipt, <-chan error)
+	Transfer(toAddr common.Address, amount *big.Int) (<-chan types.Receipt, <-chan error)
 	CurrentRoundInitialized() (bool, error)
-	Transcoder(blockRewardCut uint8, feeShare uint8, pricePerSegment *big.Int) (*types.Transaction, error)
 	IsActiveTranscoder() (bool, error)
 	TranscoderStake() (*big.Int, error)
-	Bond(amount *big.Int, toAddr common.Address) (*types.Transaction, error)
-	ValidRewardTimeWindow() (bool, error)
-	Reward() (*types.Transaction, error)
-	Job(streamId string, transcodingOptions [32]byte, maxPricePerSegment *big.Int) (*types.Transaction, error)
-	JobDetails(id *big.Int) (*big.Int, [32]byte, *big.Int, common.Address, common.Address, *big.Int, error)
-	EndJob(id *big.Int) (*types.Transaction, error)
-	ClaimWork(jobId *big.Int, startSegmentSequenceNumber *big.Int, endSegmentSequenceNumber *big.Int, transcodeClaimsRoot [32]byte) (*types.Transaction, error)
-	Verify(jobId *big.Int, segmentSequenceNumber *big.Int, dataHash [32]byte, transcodedDataHash [32]byte, broadcasterSig []byte, proof []byte) (*types.Transaction, error)
-	Transfer(toAddr common.Address, amount *big.Int) (*types.Transaction, error)
 	TokenBalance() (*big.Int, error)
-	WaitUntilNextRound(roundLength *big.Int) error
+	GetJob(jobID *big.Int) (struct {
```

Comment:
do we return this anonymous struct here just to match the smart contract return type? Does it make sense to wrap it in a type Job struct for easier use?